app/src/components/Pages/TipsAndArticles.tsx

```diff
@@ -67,6 +72,13 @@ const TipsAndArticles = (props: any) => {
     fetchTypeData("Tips");
     fetchTypeData("Articles");
   };
+  const validateRole = () => {
```

Comment:
What the function return otherwise? 
better to write:
Return role.role_type=== RoleEnum.Admin || role.role_type === RoleEnum.Psychologist